apim-operator/deploy/kubectl-extension/kubectl-add

```diff
@@ -74,9 +74,10 @@ then
         done
 
         apiName=$2
+        configmapName="$apiName-swagger"
```

Comment:
Shall we change the variable name as configMapName instead of configmapName?

Reply:
This extension is not used in the latest operator. Hence this has been removed.